sdk/anomalydetector/ai-anomaly-detector/src/AnomalyDetectorClient.ts

```diff
@@ -64,6 +64,7 @@ export class AnomalyDetectorClient {
   constructor(
     endpointUrl: string,
     credential: TokenCredential | KeyCredential,
+    // eslint-disable-next-line @azure/azure-sdk/ts-naming-options
```

Comment:
The rule seems to be this:

> require client method option parameter type names to be suffixed with Options and prefixed with the method name

I would imagine this to apply only for methods that are not constructors...